packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -201,6 +205,10 @@ export class CeramicCliUtils {
       }
       if (network) {
         config.network.name = network
+        const getIndexingConfig = getNetworkDefaultConfig(network)
+        config.indexing.allowQueriesBeforeHistoricalSync =
+          getIndexingConfig.allow_queries_before_historical_sync
```

Comment:
this is making the setting in the database (the harder one for users to change) override the behavior from the config file (the thing users actually have access to).  I think the the order of precedence for config options should be:
Env vars -> CLI flags -> config file -> database config table.

Since in this case there is no env var for cli flag, once the config file option is removed then the database table options will take over as the only way to configure this, but as long as the config file option exists it seems like it should take precedence.

Reply:
wait, this isn't even getting it from the database, this is just overriding the value from the config file with the default values all the time